tests/test_record.py

```diff
@@ -185,3 +199,34 @@ def test_dotted_name(self):
             spider.parse('pass')
             spider.record()
             spider.test()
+
+    def test_skipped_fields(self):
+        with CaseSpider() as spider:
+            spider.imports('import time')
+            spider.custom_settings('''
+                AUTOUNIT_SKIPPED_FIELDS = ['ts']
+            ''')
+            spider.start_requests("yield scrapy.Request('data:text/plain,')")
+            spider.parse('''
+                yield {'a': 4, 'ts': time.time()}
```

Comment:
As unlikely as it is, it's possible this test could pass despite being broken if `spider.test()` runs before 1ms has elapsed...

I can't think of a workaround that wouldn't be really ugly though.  Maybe add a very very short sleep here?

Reply:
Oh right. Sure, we can add a small sleep here to prevent that.